deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: pravega-operator
       containers:
         - name: pravega-operator
-          image: pravega/pravega-operator:latest
+          image: adrianmo/pravega-operator:issue-96
```

Comment:
Is it right that you want this to point to this issue/snapshot image?

Reply:
Thanks for raising this, I've reverted the image to `pravega/pravega-operator:latest`.